x-pack/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -208,34 +232,13 @@ export const signalRulesAlertType = ({
             savedId,
             services,
             index: inputIndex,
-            lists: exceptions_list,
-          });
-
-          const noReIndex = buildEventsSearchQuery({
-            index: inputIndex,
-            from,
-            to,
-            filter: esFilter,
-            size: searchAfterSize,
-            searchAfterSortId: undefined,
+            // temporary filter out list type
```

Comment:
nit: I'd do a `// TODO: temporary filter out list type until it is implemented` so we don't miss it later.